deepmd/utils/network.py

```diff
@@ -44,19 +44,31 @@ def one_layer(inputs,
                             w_initializer, 
                             trainable = trainable)
         variable_summaries(w, 'matrix')
-        b = tf.get_variable('bias', 
-                            [outputs_size], 
-                            precision,
-                            b_initializer, 
-                            trainable = trainable)
-        variable_summaries(b, 'bias')
+        if final_layer and GLOBAL_ASCEND_OUT_PRECISION:
```

Comment:
Add `is not None`
i.e. `GLOBAL_ASCEND_OUT_PRECISION is not None`

Reply:
Thanks! we have fixed it.